README.md

```diff
@@ -25,7 +25,7 @@
 
 
 
-# 📖 JavaScript Capstone Project <a name="about-project"></a>
+# 📖 Math Magicians Project <a name="about-project"></a>
```

Comment:
- [ ] All components should be tested. Kindly add tests for the `src/App.js` component as requested by the previous reviewer.

Reply:
Thank you @TedLivist  but it was an optional suggestion. Because of this reason, I asked second opinion.